data/translations/kdiskmark_nl_NL.ts

```diff
@@ -0,0 +1,509 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="nl_NL">
+<context>
+    <name>About</name>
+    <message>
+        <location filename="../../src/about.ui" line="17"/>
+        <source>About KDiskMark</source>
+        <translation>Over KDiskMark</translation>
+    </message>
+    <message>
+        <location filename="../../src/about.ui" line="62"/>
+        <source>Version:</source>
+        <translation>Versie:</translation>
+    </message>
+    <message>
+        <location filename="../../src/about.ui" line="88"/>
+        <source>License:</source>
+        <translation>Licentie:</translation>
+    </message>
+    <message>
+        <location filename="../../src/about.ui" line="114"/>
+        <source>Author:</source>
+        <translation>Ontwikkelaar:</translation>
+    </message>
+    <message>
+        <location filename="../../src/about.ui" line="146"/>
+        <source>E-mail:</source>
+        <translation>E-mailadres:</translation>
+    </message>
+    <message>
+        <location filename="../../src/about.ui" line="178"/>
+        <source>Flexible I/O Tester:</source>
+        <translation>Flexibele I/O-tester:</translation>
```

Comment:
This is the name of the package. It shouldn't be translated. It is possible to translate it for languages which may have differences like spaces before colons.